File: C++/Estrutura Repetitiva/Para/Par_impar.cpp
#include <iostream>
#include <iomanip>
#include <string>
#include <climits>
#include <cmath>

using namespace std;

int main()
{

    int N, i, x;

    cout << "Quantos numeros voce vai digitar? ";
    cin >> N;

    for (i = 1; i <= N; i++)
    {
        cout << "Digite um numero: ";
        cin >> x;
        if (x % 2 == 0 && x != 0)
        {
            if (x > 0)
            {
                cout << "PAR POSITIVO" << endl;
            }
            else
            {
                cout << "PAR NEGATIVO" << endl;
            }
        }
        else if (x % 2 != 0 && x != 0)
        {
            if (x > 0)
            {
                cout << "IMPAR POSITIVO" << endl;
            }
            else
            {
                cout << "IMPAR NEGATIVO" << endl;
            }
        }
        else
        {
            cout << "NULO" << endl;
        }
    }

    return 0;
}

File: Python/Estrutura Repetitiva/Para/Par_impar.py
N: int; i: int; x: int

N = int(input("Quantos numeros voce vai digitar? "))

for i in range (1, N + 1):
    x = int(input("Digite um numero: "))
    if x % 2 == 0 and x != 0:
        print(f"PAR ", end="")
    elif x % 2 != 0 and x != 0:
        print(f"IMPAR ", end="")
    elif x == 0:
        print(f"NULO")

    if x > 0:
        print(f"POSITIVO")
    elif x < 0:
        print(f"NEGATIVO")

            